quadlet/redpanda/redpanda.container

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=A RedPanda single node development cluster
+After=network-online.target
+
+[Container]
+Image=docker.redpanda.com/redpandadata/redpanda:latest
+ContainerName=redpanda-server
+Exec=redpanda start --overprovisioned --smp 1 --kafka-addr internal://0.0.0.0:9092,external://0.0.0.0:19092 --advertise-kafka-addr internal://redpanda-server:9092,external://localhost:9092 --pandaproxy-addr internal://0.0.0.0:8082,external://0.0.0.0:18082 --advertise-pandaproxy-addr internal://redpanda-server:8082,external://localhost:18082 --schema-registry-addr internal://0.0.0.0:8081,external://0.0.0.0:18081 --rpc-addr redpanda-server:33145 --advertise-rpc-addr redpanda-server:33145
```

Comment:
I know it's ugly, but I couldn't figure out a better way.

Reply:
I don't know redpanda. But, can't you pass all this information using a config file?

Reply:
Yes, I believe it can be. Will try switching these flags over to a config file.

Reply:
While they do support configuring the server with a config file, I am running into issues which I believe are due to mismatching file permissions.  Will keep looking into this.  
I should be able to use the UIDMap and GIDMap, but just need to figure out the expected values.

Reply:
Blocked on:
https://github.com/redpanda-data/redpanda/issues/15902

Redpanda tries to overwrite the config file with another, but you cannot replace a mounted file with another - you need to replace it.  I'm sure there's some hackery I can do, but would rather wait and have it fixed upstream

Reply:
Have you tried mounting the directory instead of the file? Maybe move the file into a dedicated directory.